src/main/ts/components/Editor.tsx

```diff
@@ -86,9 +82,10 @@ export class Editor extends React.Component<IAllProps> {
       ScriptLoader.load(
         this.elementRef.current.ownerDocument,
         this.getScriptSrc(),
-        this.initialise,
-        this.props.loadTimeOut,
-        this.props.scriptAttributes
+        this.props.scriptLoading?.async ?? false,
```

Comment:
oh is this some new way of defaulting optional args

Reply:
See here:
https://mariusschulz.com/blog/nullish-coalescing-the-operator-in-typescript

Reply:
```
value ?? defaultValue
```
is equivalent to:
```
value !== null && value !== undefined ? value : defaultValue
```